routes/api-routes.js

```diff
@@ -0,0 +1,53 @@
+const express = require("express");
+const router = express.Router();
+const fs = require("fs");
+const uniqid = require("uniqid");
+
+router.get("/api/notes", (req, res) => {
+  // utf8 format because this is a html format
+  fs.readFile("db/db.json", "utf8", (err, data) => {
+    //   This is different to console.log error
+    if (err) {
+      console.log(err);
+    } else {
+      console.log("api");
+      return res.json(JSON.parse(data));
+    }
+  });
+});
+
+router.post("/api/notes", (req, res) => {
+  fs.readFile("db/db.json", "utf8", (err, data) => {
+    if (err) throw err;
+    const noteDb = JSON.parse(data);
+    const newNote = req.body;
+    const noteId = uniqid();
+    const id = "id";
+    newNote[id] = noteId;
+    noteDb.push(newNote);
```

Comment:
Great use of the .push array method :-)
As you know, this method adds the item to the end of your array.
If you wanted to, you could use .unshift() instead which will place the item at the beginning of the array - this would mean that when your notes render in the UI the newest note will always be at the top :)